internal/auth/jwt/jwt.go

```diff
@@ -0,0 +1,49 @@
+package jwt
+
+import (
+	"errors"
+	"os"
+	"time"
+
+	"github.com/golang-jwt/jwt/v5"
+)
+
+var jwtKey = []byte(os.Getenv("JWT_SECRET_KEY"))
```

Comment:
Does this key have to be something specific?

Reply:
Yes, I generated a key using `openssl rand -base64 32` stored it as an environment variable.

Reply:
So would you need to send me this key so it's consistent among our machines? We'd also have to add it to our app service right?

Reply:
yes